README.md

```diff
@@ -1,44 +1,58 @@
--* Install a Git Client
--    * Here's [a nice GUI client](https://desktop.github.com).
--    * Here's [a standard client](https://git-scm.com).
--
--Once you have the prerequisites installed, you can install the Aurelia CLI itself. From the command line, use npm to install the CLI globally:
--
--```
--npm install aurelia-cli -g
--```
--
--> Note: Always run commands from a Bash prompt. Depending on your environment, you may need to use `sudo` when executing npm global installs.
--
--## Application Setup
--
--Once you've setup your machine (or if it's been previous setup), you simply need to install the dependencies. From within the app folder, execute the following command:
--
--```
--npm install
--```
--
--## Running The App
--
--To run the app, execute the following command from within the app folder:
--
--```
--au run --env dev
--```
--
--See [the CLI documentation](https://github.com/aurelia/cli) for other available commands or type `au help` on the command line.
+# app-ux-showcase
+
+[![ZenHub](https://raw.githubusercontent.com/ZenHubIO/support/master/zenhub-badge.png)](https://zenhub.io)
+[![Join the chat at https://gitter.im/aurelia/discuss](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/aurelia/discuss?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+An application that showcases the various features of Aurelia UX.
+
+## Machine Setup
+
+This app is built on the Aurelia CLI, which has a couple of prerequisites you must install first. If you have previously setup your machine for the Aurelia CLI, you can skip this step.
+
+* Install NodeJS >= 4.x
+    * You can [download it here](https://nodejs.org/en/).
+* Install NPM 3.x
+    * Even though you may have the latest NodeJS, that doesn't mean you have the latest version of NPM. You can check your version with `npm -v`. If you need to update, run `npm install npm -g`.
+* Install a Git Client
+    * Here's [a nice GUI client](https://desktop.github.com).
+    * Here's [a standard client](https://git-scm.com).
+
+Once you have the prerequisites installed, you can install the Aurelia CLI itself. From the command line, use npm to install the CLI globally:
+
+```
+npm install aurelia-cli -g
+```
+
+> Note: Always run commands from a Bash prompt. Depending on your environment, you may need to use `sudo` when executing npm global installs.
+
+## Application Setup
+
+Once you've setup your machine (or if it's been previous setup), you simply need to install the dependencies. From within the app folder, execute the following command:
+
+```
+npm install
+```
+
+## Running The App
+
+To run the app, execute the following command from within the app folder:
+
+```
+au run --env dev
+```
+
+See [the CLI documentation](https://github.com/aurelia/cli) for other available commands or type `au help` on the command line.
+
+```
+npm install
```

Comment:
Information starting at line 33 is repeated here. I am guessing this is a mistake?

Reply:
It's difficult to see with the diff, but the README.md has a `-` on just about every single line. There's nothing wrong with the information - just the formatting.

### Before:
![aurelia-app-ux-showcase](https://user-images.githubusercontent.com/8031689/35097645-a4d592d8-fc16-11e7-82f9-1e22283443d9.png)

### After:
![sethbergman-app-ux-showcase-at-update-readme](https://user-images.githubusercontent.com/8031689/35098073-633e75d6-fc18-11e7-82a0-7332d33e013e.png)

Hopefully this helps clarify the issue. :smiley:

Reply:
Ohh wait. Now I see what you're saying. Yes, let me get that fixed. :laughing: